### PR TITLE
chore(button-next): add affix icon data hooks

### PR DESCRIPTION
**Pull request by omeb**
Following this PR - https://github.com/wix/wix-style-react/pull/6831

This PR contains the following:
- add data hooks for button icon affixes
- expose methods that verifies affix icon existence in `unidriver`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "devDependencies": {
-    "lerna": "^2.8.0"
+    "lerna": "^3.22.1"
```
Comment (mykas) [inline package.json:3]: redudant change. This repo is not monorepo, but n+1 package.

File: packages/wix-ui-core/src/components/button-next/button-next.uni.driver.ts [hunk 2]
```diff
@@ -3,6 +3,9 @@ import {
   baseUniDriverFactory,
```
Comment (mykas) [inline packages/wix-ui-core/src/components/button-next/button-next.uni.driver.ts:3]: Missing same implementation in driver.ts too

File: package.json (hunk 1 above)
Comment (omeb) [reply package.json:3]: Agreed, removed 👍🏼

File: packages/wix-ui-core/src/components/button-next/button-next.uni.driver.ts (hunk 2 above)
Comment (omeb) [reply packages/wix-ui-core/src/components/button-next/button-next.uni.driver.ts:3]: sorry, which `driver.ts` are you referring to? 
[This one](https://github.com/wix/wix-ui/blob/master/packages/wix-ui-core/src/components/button-next/button-next.driver.ts#L1) imports the unidriver itself, and [this one](https://github.com/wix/wix-ui/blob/master/packages/wix-ui-core/src/components/button-next/button-next.driver.private.ts) is an internal one for the internal tests...

**Review by mykas — APPROVED** [review]

**Review by mykas — APPROVED** [review]

**Pull request merged**